<launch>
    <!-- <rosparam param="cmd_ratio">1.0</rosparam>
    <rosparam param="RL_model">1</rosparam> -->
    <arg name="use_rviz" default="true" />
    
    <param name="/use_sim_time" value="true"/>
    <param name="robot_description" textfile="$(find ur5_pybullet_ros)/script/urdf/ur5_robotiq_85_mobile_rviz.urdf"/>
    
    <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
        <param name="use_tf_static" value="true"/>
    </node>

    <node pkg="ur5_pybullet_ros" type="ur5_demo.py" name="ur5_pybullet" output="screen"/>

    <include file="$(find navi_slam)/launch/navi_amcl.launch"/> 
    <!-- <node pkg="navi_slam" type="learning_navi.py" name="learning_navi" respawn="true" output="screen"/> -->

    <group if="$(arg use_rviz)">
         <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ur5_pybullet_ros)/config/ur5.rviz"  respawn="false" />
    </group>

</launch>
